### PR TITLE
Tuesday March 2 2021 Meeting

### PR DESCRIPTION
**Pull request by nkohen**
Please comment on this PR or DM with things we should discuss or other changes that should be made to this document.

Once the meeting has happened I will add meeting notes and then once they have been reviewed this will be merged into meetings directory.

### CODE REVIEW

File: meeting/meeting_march_2_2021.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/142
+* Merged Disjoint Union DLC spec
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/143
+* Opened PR proposing `1/x` payout curve shape support
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/144
+* Fraud proof discussion on mailing list
+  * https://mailmanlists.org/pipermail/dlc-dev/2021-February/000020.html
+* Support for non-corresponding multi-oracle DLCs discussion on mailing list
+  * https://mailmanlists.org/pipermail/dlc-dev/2021-February/000024.html
+
```
Comment (benthecarman) [inline meeting/meeting_march_2_2021.md:84]: mailing list talk about oracle time stamps too

**Review by Tibo-lg — COMMENTED** [review]
Maybe add mentions to #141 #132 #134

File: meeting/meeting_march_2_2021.md (hunk 1 above)
Comment (nkohen) [reply meeting/meeting_march_2_2021.md:84]: https://github.com/discreetlogcontracts/dlcspecs/pull/148/files#diff-9ae033a19a006f525d653236287da0be3a94d9db98e0b3e14d9a51d8d287305eR70

File: meeting/meeting_march_2_2021.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+* TODO
+
+## Oracle Specifications
+
+**#Status_Update**
+
+* [Oracle Event Timestamps Discussion](https://mailmanlists.org/pipermail/dlc-dev/2021-February/000015.html)
+  * Current proposal is to replace u32 timestamp on `oracle_event` with a TLV which can either be `u32 : expected_time` or `u32 : earliest_expected_time, u32 : latest_expected_time`.
+  * Does anyone think it is ever useful for oracles to give any timestamps other than expected times?
```
Comment (ariard) [inline meeting/meeting_march_2_2021.md:101]: Yeah see my contra opinion on the ML :) ?

File: meeting/meeting_march_2_2021.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+* [Oracle Attestation Computation Change Proposal](https://mailmanlists.org/pipermail/dlc-dev/2020-December/000002.html)
+  * **#Query(Tibo, Lloyd)**
+  * **#Discussion**
+
```
Comment (ariard) [inline meeting/meeting_march_2_2021.md:105]: May we add #120 ? Finally updated with latest spec changes

Comment (ariard) [inline meeting/meeting_march_2_2021.md:105]: We should also talk about initial discovery of oracles by DLC clients, see this [discussion](https://github.com/discreetlogcontracts/dlcspecs/pull/120#discussion_r538986097)

**Pull request merged**